test/errors/laterVar-brief.good

```diff
@@ -0,0 +1 @@
+error in laterVar.chpl:1: statement references a variable before it is defined
```

Comment:
Would it be possible to have the brief mode show the definition point as well?  I think it's common for a user to think they're referring to an earlier variable, only to be caught by surprise by another declaration (potentially much) later in the code.  Pointing out where the declaration is coming from prevents them from having to find it manually.